cilium/cmd/bpf_vtep_list.go

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2018 Authors of Cilium
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/pkg/common"
+	"github.com/cilium/cilium/pkg/maps/vtep"
+)
+
+const (
+	vtepCidrTitle = "IP PREFIX/ADDRESS"
+	vtepTitle     = "VTEP"
+)
+
+var (
+	ciliumMapGet  = "you may run cilium map get cilium_vtep_map.\n"
+	vtepListUsage = "List VTEP CIDR and their corresponding VTEP MAC/IP.\n"
+)
+
+var bpfVtepListCmd = &cobra.Command{
+	Use:     "list",
+	Aliases: []string{"ls"},
+	Short:   "List List VTEP CIDR and their corresponding VTEP MAC/IP",
+	Long:    vtepListUsage,
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("cilium bpf vtep list")
+
+		bpfVtepList := make(map[string][]string)
+		if err := vtep.VtepMAP.Dump(bpfVtepList); err != nil {
+			fmt.Fprintf(os.Stderr, "error dumping contents of map: %s\n", err)
+			os.Exit(1)
+		}
+
+		if command.OutputJSON() {
+			if err := command.PrintOutput(bpfVtepList); err != nil {
+				fmt.Fprintf(os.Stderr, "error getting output of map in JSON: %s\n", err)
+				os.Exit(1)
+			}
+			return
+		}
+
+		if len(bpfVtepList) == 0 {
+			fmt.Fprintf(os.Stderr, "No entries found.\n%s\n", ciliumMapGet)
```

Comment:
Do you need a specific variable for the second part of the message? It looks like you're not reusing it later?
Also you have a double line break (in format string _and_ in the string in the variable).